rights-statements.ttl

```diff
@@ -28,202 +30,219 @@
 
 <collection-ic> a skos:Collection ;
```

Comment:
* Should the collection URIs be versioned? I think they at least need trailing slashes.
* Should collections themselves be versioned (with `owl:versionInfo`?)